webapp/components/profile_picture.jsx

```diff
@@ -25,21 +28,96 @@ export default class ProfilePicture extends React.Component {
     }
 
     render() {
+        let email = '';
+        let profileImg = '';
+        let toRender;
         let statusClass = '';
         if (this.props.status) {
             statusClass = 'status-' + this.props.status;
         }
-
-        return (
-            <span className={`status-wrapper ${statusClass}`}>
+        if (this.props.user) {
+            email = this.props.user.email;
+            profileImg = Client.getUsersRoute() + '/' + this.props.user.id + '/image?time=' + this.props.user.update_at;
```

Comment:
Unless I'm missing something, we actually already have the profile image URL in `this.props.src`. Would you be able to use that here instead?